examples/digits.jl

```diff
@@ -8,13 +8,13 @@ using StableRNGs
 using Flux: onehotbatch, onecold, crossentropy, throttle
 using Base.Iterators: repeated, partition
 using Legolas, LegolasFlux
+using Tables
 
 # This should store all the information needed
 # to construct the model.
-Base.@kwdef struct DigitsConfig
-    seed::Int = 5
-    dropout_rate::Float32 = 0f1
-end
+const DigitsConfig = Legolas.@row("digits-config@1",
```

Comment:
Changed to a Legolas object so it serializes easier (otherwise need manual Arrow stuff)

Reply:
Hm, does legolas pre 0.4 support this?  Looks like it's deserializing as a NamedTuple.